docs/reference/ingest/apis/simulate-pipeline.asciidoc

```diff
@@ -158,15 +154,14 @@ POST _ingest/pipeline/_simulate?verbose
   "docs": [
     {
       "_index": "index",
-      "_type": "_doc",
       "_id": "id",
       "_source": {
         "foo": "bar"
       }
     },
     {
       "_index": "index",
-      "_type": "_doc",
+      "_type": "_type",
```

Comment:
I would go ahead and remove the `_type` example here as well. We don't to encourage users to do anything with it.

Reply:
This is an example of the output of the call, which will have the `_type` field present.  However, it looks like we can just provide `null` for the type to the IngestDocument and `_type` will be omitted from the response - what do you think of doing that instead of changing the default to `_doc`?

I've removed `_type` from all of the example inputs already.

Reply:
> However, it looks like we can just provide null for the type to the IngestDocument if one isn't provided in the request and _type will be omitted from the response - what do you think of doing that instead of changing the default to _doc

I think we should keep the `"_type": "_doc"` on the return values since it better mirrors the return of a typeless search

```
POST foo/_create/1
{}
GET foo/_search
```

Also, this line (164) is from the request, and there are some responses with it missing (for example 186-187).